<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProjectView</class>
 <widget class="QWidget" name="ProjectView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>280</width>
    <height>260</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="lineEditSearch">
       <property name="placeholderText">
        <string>Filter</string>
       </property>
       <property name="clearButtonEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonImport">
       <property name="text">
        <string>Import</string>
       </property>
       <property name="icon">
        <iconset resource="../gameplay-editor.qrc">
         <normaloff>:/res/images/file-import.png</normaloff>:/res/images/file-import.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>18</width>
         <height>18</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="ProjectTreeView" name="treeView">
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
     <property name="uniformRowHeights">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
  <action name="actionOpenFile">
   <property name="text">
    <string>Open File</string>
   </property>
  </action>
  <action name="actionNewFile">
   <property name="text">
    <string>New File ...</string>
   </property>
  </action>
  <action name="actionNewFolder">
   <property name="text">
    <string>New Folder ...</string>
   </property>
  </action>
  <action name="actionImportFile">
   <property name="text">
    <string>Import File ...</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ProjectTreeView</class>
   <extends>QTreeView</extends>
   <header>ProjectTreeView.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../gameplay-editor.qrc"/>
 </resources>
 <connections/>
 <slots>
  <slot>openFile()</slot>
 </slots>
</ui>
